align.go

```diff
@@ -35,6 +33,7 @@ type TextQualifier struct {
 type PaddingOpts struct {
 	Justification  Justification
 	ColumnOverride map[int]Justification //override the Justification of specified columns
+	Surround       int                   // additional padding surrounding separator
```

Comment:
Any reason not to use `Padding int`?

Reply:
I think that I was heading down another path when I added that.  Fixed now.